content/docs-v1/quire-commands.md

```diff
@@ -30,17 +30,20 @@ Quire commands are typically run from Terminal on a Mac and PowerShell on a PC.
 
 | Command       | Action   | Notes              |
 | ------------- | -------- | ------------------ |
-| **`quire build`** | Build site files used to create various outputs | This creates the site files for the HTML format of the project. These site files are then used to create the PDF or EPUB versions of the project. This is therefore a necessary step before running the commands `quire epub` and `quire pdf`.|
-| **`quire pdf`** | Output Quire project in PDF format | You must run `quire build` before running this command. |
-| **`quire epub`** | Output Quire project in EPUB format | You must run `quire build` before running this command. |
-| **`quire clean`** | Remove old build outputs | |
+| **`quire build`** | Build Quire publication outputs | This creates the site files for the HTML format of the project. These site files are then used to create the PDF or EPUB versions of the project. This is therefore a necessary step before running the commands `quire epub` and `quire pdf`.|
+| **`quire pdf`** | Build Quire project in PDF format | You must run `quire build` before running this command. |
+| **`quire epub`** | Build Quire project in EPUB format | You must run `quire build` before running this command. |
+| **`quire site`** | Build Quire project in HTML format. | This is similar to running `quire build` command. |
```

Comment:
`quire site` isn't a working command

Reply:
OK I was confused about that one. Found it in the `readme`. Thank you!